src/Proxy/Autoloader.php

```diff
@@ -20,7 +20,7 @@
 /**
  * Special Autoloader for Proxy classes, which are not PSR-0 compliant.
  *
- * @internal
+ * @deprecated The Autoloader class is deprecated since doctrine/common 3.5.
```

Comment:
Deprecating the class does not make it not internal. 😬

Reply:
fix :see_no_evil: